<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Boincmanager</class>
 <widget class="QMainWindow" name="Boincmanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>497</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BOINC @ NITC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>671</width>
      <height>421</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QPushButton" name="start">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>26</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start BOINC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop BOINC</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>501</width>
        <height>371</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Projects</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>130</y>
        <width>581</width>
        <height>251</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="createac">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Create Account</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter Project URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prurl1">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>331</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="email">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>331</width>
          <height>27</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>71</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>User name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="uname">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>331</width>
          <height>27</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>15</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pwd">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>130</y>
          <width>331</width>
          <height>27</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
        </property>
        <property name="maxLength">
         <number>12</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="createac_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>180</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="attachpro">
       <attribute name="title">
        <string>Attach / Detach Project</string>
       </attribute>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter Project URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prurl2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>371</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Authentication Token</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="auth">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>90</y>
          <width>371</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="attach">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>160</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Attach</string>
        </property>
       </widget>
       <widget class="QPushButton" name="detach">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>160</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Detach</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>Resume / Suspend Project</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Project URL</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="prurl3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>341</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="reset">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
       <widget class="QPushButton" name="update">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
       <widget class="QPushButton" name="resume">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>140</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Resume</string>
        </property>
       </widget>
       <widget class="QPushButton" name="suspend">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>140</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Suspend</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTextEdit" name="projectext">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>27</y>
        <width>561</width>
        <height>81</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>641</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Disk Usage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Domain Name :</string>
      </property>
     </widget>
     <widget class="QLabel" name="domname">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>260</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Operating System :</string>
      </property>
     </widget>
     <widget class="QLabel" name="os">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>260</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CPU Model :</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>531</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Memory Size :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cache Size :</string>
      </property>
     </widget>
     <widget class="QLabel" name="memory">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>320</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cache">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CPU FP OPS  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="fp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>350</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CPU int OPS :</string>
      </property>
     </widget>
     <widget class="QLabel" name="int_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>350</y>
        <width>231</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>641</width>
        <height>151</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Project Status</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Notices</string>
     </attribute>
     <widget class="QTextEdit" name="notices">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>651</width>
        <height>361</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextBrowserInteraction</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BOINC @ NIT CALICUT</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
